nservicebus/recoverability/index.md

```diff
@@ -100,3 +100,9 @@ The output in the log will be:
 snippet: RetryLogging
 
 partial: exceptionincluded
+
+
+## Recoverability memory consumption
+
+MSMQ and SQL Server transport need to cache exceptions in memory for retries. Therefore, exceptions with a large memory footprint can cause high memory usage of the NServiceBus process. NServiceBus can cache up to 1000 exceptions, capping the potential memory consumption to 1000 x `<exception size>`. Please refer to [this guide](/nservicebus/recoverability/lru-memory-consumption.md) to resolve problems due to execessive memory consumption.
```

Comment:
@bording @SeanFeldman looking at the code we couldn't see that ASB is using a cache for the exceptions. Can you confirm that we haven't missed anything? (given that it does have support for atomic sends with receive)

Reply:
That's correct. ASB doesn't.